src/dso_api/dynamic_api/temporal.py

```diff
@@ -173,14 +173,15 @@ def _filter_queryset(self, queryset: models.QuerySet, prefix: str):
             # and allow ?geldigOp=* to return all data
             return queryset
 
+        identifier = self.table_schema.identifier[0]  # from ["identificatie", "volgnummer"]
```

Comment:
Dit is potentieel gevaarlijk, omdat we nergens de volgorde van het `.identifier` veld afdwingen. Het zou dus ook `["volgnummer", "identifier"]` kunnen zijn. Maar, `table_schema` heeft een attribuut `temporal`. Dat is een object dat ook weer een `identifier` attribuut heeft. Dat is het attribuut dat een temporal object uniek maakt. In ons geval is dat `volgnummer`. Dus, in code gaat het ons om `set(self.table_schema.identifier) - {self.table_schema.temporal.identifier}` ofwel `{'identificatie', 'volgnummer'} - {'volgnummer'} = {'identificatie'}`.

Reply:
Deze strategie wordt op meerdere plekken gebruikt. Voor dit stuk heb ik in ieder geval je advies opgevolgd en de identifier expliciet extracted.